packages/graphics/src/image.ts

```diff
@@ -251,6 +257,10 @@ Image.prototype.rotated = function (deg: number): Image {
     }
 }
 
+Image.prototype.getContext = function (): ImageRenderingContext {
+    return new ImageRenderingContext(this)
```

Comment:
we should either rename this `createContext()` or have it return the same value after construction as the web version does...

Reply:
(I would be inclined to rename, more flexible and memory efficient)

Reply:
I can, name?